### PR TITLE
Feature/multiple characters

### PR DESCRIPTION
**Pull request by JoepEilander**
added 6 playable character models and textures

### CODE REVIEW

**Review by TryFox — CHANGES_REQUESTED** [review]
Please fix the issue with the idle animation not working

**Review by TryFox — APPROVED** [review]
Everything works now. Approved all changes

**Review by lucasvdiepen — CHANGES_REQUESTED** [review]
A lot of texture files are missing a suffix.

**Review by lucasvdiepen — APPROVED** [review]

**Review by Feetjexx — APPROVED** [review]
Approved

File: Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs [hunk 1]
```diff
@@ -220,5 +219,14 @@ private void DespawnPlayerOnServer(NetworkObject networkObject)
         {
             ServerManager.Despawn(networkObject);
         }
+
+        private GameObject GetAndRemoveRandomPlayerPrefab()
```
Comment (PiterGroot) [inline Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs:221]:
```suggestion
        [Server]
        private GameObject GetAndRemoveRandomPlayerPrefab()
```

File: Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs [hunk 2]
```diff
@@ -220,5 +219,14 @@ private void DespawnPlayerOnServer(NetworkObject networkObject)
         {
             ServerManager.Despawn(networkObject);
```
Comment (PiterGroot) [inline Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs]: Make one liner

File: Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs [hunk 3]
```diff
@@ -220,5 +219,14 @@ private void DespawnPlayerOnServer(NetworkObject networkObject)
...
         }
+
+        private GameObject GetAndRemoveRandomPlayerPrefab()
+        {
+            int randomIndex = Random.Range(0, _availablePlayerPrefabs.Count);
+            GameObject randomPlayerPrefab = _availablePlayerPrefabs[randomIndex];
+            _availablePlayerPrefabs.RemoveAt(randomIndex);
```
Comment (PiterGroot) [inline Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs:226]:
```suggestion

            _availablePlayerPrefabs.RemoveAt(randomIndex);
```

**Review by PiterGroot — CHANGES_REQUESTED** [review]
Nice work. All players are white, maybe because of the texture renaming. Also don't forget to update from develop.

**Review by PiterGroot — APPROVED** [review]

**Pull request merged**